cloudformation-telemetry/lambda_function.py

```diff
@@ -61,31 +60,37 @@ def telemetryFactory(event, context):
 
 # Interface
 class baseTelemetry(ABC):
+
+    def __init__(self, event,context,*args, **kwargs):
+        self.event=event
+        self.context=context
+        self.config = read_yaml_file("./metadata.yaml")
+        self.config['SumoLogic']['SUMO_ENDPOINT'] = self.event['ResourceProperties']['TelemetryEndpoint']
+        self.sumoHttpHandler = HTTPHandler(self.config)
+        self.log = self.sumoHttpHandler.log
+
     @abstractmethod
     def fetch_and_send_telemetry(self):
         raise NotImplementedError
 
-    # Use clientMixing func from app-client-sdk
-    def send_telemetry(self, data, endpoint):
-        headers = {'content-type': 'application/json'}
-        print("Telemetry enabled")
-        r = requests.post(endpoint, data = json.dumps(data), headers=headers)
+    def send_telemetry(self, data):
+        self.log.debug("Telemetry enabled") # replace at all places
```

Comment:
this enabled message we can shift to init otherwise everytime send telemetry is called everytime this log line would be printed.